environment-api/src/main/java/com/sequenceiq/environment/api/doc/environment/EnvironmentModelDescription.java

```diff
@@ -41,6 +41,7 @@
     public static final String CREATE_FREEIPA = "Create freeipa in environment";
     public static final String FREEIPA = "The FreeIPA paramaters";
     public static final String FREEIPA_INSTANCE_COUNT_BY_GROUP = "The number of FreeIPA instances to create per group when creating FreeIPA in environment";
+    public static final String MULTIAZ_FREEIPA = "The FreeIPA multi az";
```

Comment:
Nit: Does not matter much, perhaps `multi az` should be rather spelled as `multi-AZ`.